compiler/driver/src/argparser.rs

```diff
@@ -82,16 +82,16 @@ fn compile_command<'a, 'b>() -> App<'a, 'b> {
         .about("Compiles Erlang sources to an executable or shared library")
         .setting(AppSettings::DeriveDisplayOrder)
         .arg(
-            Arg::with_name("input")
+            Arg::with_name("inputs")
                 .index(1)
                 .help(
-                    "Path to the source file or directory to compile.\n\
+                    "Path(s) to the source file(s) or director(y|ies) to compile.\n\
                      You may also use `-` as a file name to read a file from stdin.\n\
                      If not provided, the compiler will use the current directory as input.",
                 )
                 .next_line_help(true)
-                .takes_value(true)
-                .value_name("PATH"),
+                .multiple(true)
```

Comment:
Are we sure this does the right thing? The way the `raw` argument is configured, I would expect some kind of conflict, as I'm not sure how it resolves the two arguments, unless it requires `--` for `raw` due to the use of `last(true)`

Reply:
The output looks right for the help:

```
lumen compile [FLAGS] [OPTIONS] [PATHS]... [-- <ARGS>...]
```

But, yes, you need `--` to separate the LLVM args, but the old version showed `lumen compile [FLAGS] [OPTIONS] [PATH. [-- <ARGS>...]`, so I don't think `--` is a new requirement.